docs/source/internals/rest_api.rst

```diff
@@ -19,10 +19,11 @@ Extending the REST API
 ======================
 
 In the following, we will go through a minimal example of creating an API that extends the AiiDA REST API by adding an endpoint ``/new-endpoint``.
-The endpoint will support two HTTP methods:
+The endpoint implements a ``GET`` request that retrieves the latest created ``Dict`` node and returns its ``id``, ``ctime`` in ISO 8601 format, and ``attributes``.
 
- * *GET*: retrieves the latest created Dict object and returns its ``id``, ``ctime`` in ISO 8601 format, and ``attributes``.
- * *POST*: creates a ``Dict`` object with placeholder attributes, stores it, and returns its ``id``.
+.. warning::
+
+    The REST API is read-only and does not support end-points that create new data or mutate existing data in the database.
```

Comment:
please add a link to a place where people can find more info, e.g. to aiida-restapi or to the AEP
```suggestion
    The REST API is currently read-only and does not support end-points that create new data or mutate existing data in the database.
    See `this AiiDA enhancement proposal draft <https://github.com/aiidateam/AEP/pull/24>`_ for efforts in this direction.
```